src/base/bittorrent/session.cpp

```diff
@@ -62,10 +62,6 @@ using namespace BitTorrent;
 #include <libtorrent/extensions/smart_ban.hpp>
 //#include <libtorrent/extensions/metadata_transfer.hpp>
 
-#ifndef DISABLE_COUNTRIES_RESOLUTION
-#include "base/net/geoipmanager.h"
```

Comment:
If you mean this place, then I just deleted it here as it useless in session.cpp.
If you mean that we shouldn't use DISABLE_COUNTRIES_RESOLUTION now I say No. We just need to add appropriate configure option. Someone may want to turn it off to build more compact binary.